browser-extension/styles/inject.css

```
@@ -0,0 +1,244 @@
+        transition: none !important;
+    }
+
+    .mediafusion-upload-btn .mediafusion-dot {
+        animation: mediafusion-fade 2s ease-in-out infinite !important;
+        transform: scale(1) !important;
+    }
+}
+
+@keyframes mediafusion-fade {
+    0%, 100% {
+        opacity: 0.6 !important;
```

Biome (2.1.2)

[error] Using !important within keyframes declaration is completely ignored in some browsers.

Consider removing useless !important declaration.

(lint/suspicious/noImportantInKeyframe)